fastlane/Fastfile

```diff
@@ -543,58 +595,56 @@ platform :ios do
         contact_phone: ENV["APPLE_CONTACT_PHONE"],
         demo_account_name: ENV["APPLE_DEMO_EMAIL"],
         demo_account_password: ENV["APPLE_DEMO_PASSWORD"],
-        notes: "1. In the Expensify app, enter the email 'appletest.expensify@proton.me'. This will trigger a sign-in link to be sent to 'appletest.expensify@proton.me'
-                2. Navigate to https://account.proton.me/login, log into Proton Mail using 'appletest.expensify@proton.me' as email and the password associated with 'appletest.expensify@proton.me', provided above
-                3. Once logged into Proton Mail, navigate to your inbox and locate the email triggered in step 1. The email subject should be 'Your magic sign-in link for Expensify'
-                4. Open the email and copy the 6-digit sign-in code provided within
-                5. Return to the Expensify app and enter the copied 6-digit code in the designated login field"
+        notes: "1. In the Expensify app, enter the email 'appletest.expensify@proton.me'. This will trigger a sign-in link to be sent to 'appletest.expensify@proton.me'\n                2. Navigate to https://account.proton.me/login, log into Proton Mail using 'appletest.expensify@proton.me' as email and the password associated with 'appletest.expensify@proton.me', provided above\n                3. Once logged into Proton Mail, navigate to your inbox and locate the email triggered in step 1. The email subject should be 'Your magic sign-in link for Expensify'\n                4. Open the email and copy the 6-digit sign-in code provided within\n                5. Return to the Expensify app and enter the copied 6-digit code in the designated login field"
```

Comment:
NAB but why this change?

Reply:
I just applied a bunch of spacing fixes on this file